cEP-0018.md

```diff
@@ -22,6 +22,9 @@ they can write advanced linting bears. This will aim at supporting the more
 flexible visitor based method of AST traversal (as opposed to listener based
 mechanisms).
 
+Since ANTLR has poor support for retrieving original source code from the
+parse tree, this library will not be able to support fixes for code.
```

Comment:
this library will not be able to support fixes for code. -> The feature to fix code will be planned for future support.

Reply:
Or

> parse tree, the feature to fix code is out of scope for this project, and will be proposed as a future project.